src/components/CountryPicker/index.js

```diff
@@ -0,0 +1,94 @@
+import React, {useEffect, useState} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import styles from '../../styles/styles';
+import MenuItemWithTopDescription from '../MenuItemWithTopDescription';
+import useLocalize from '../../hooks/useLocalize';
+import CountrySelectorModal from './CountrySelectorModal';
+import FormHelpMessage from '../FormHelpMessage';
+
+const propTypes = {
+    /** Form Error description */
+    errorText: PropTypes.string,
+
+    /** Country to display */
+    value: PropTypes.string,
+
+    /** Callback to call when the input changes */
+    onInputChange: PropTypes.func,
+
+    /** A ref to forward to MenuItemWithTopDescription */
+    forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.instanceOf(React.Component)})]),
+};
+
+const defaultProps = {
+    value: undefined,
+    forwardedRef: undefined,
+    errorText: '',
+    onInputChange: () => {},
+};
+
+function CountryPicker({value, errorText, onInputChange, forwardedRef}) {
+    const {translate} = useLocalize();
+    const allCountries = translate('allCountries');
+    const [isPickerVisible, setIsPickerVisible] = useState(false);
+    const [searchValue, setSearchValue] = useState(lodashGet(allCountries, value, ''));
+
+    useEffect(() => {
+        setSearchValue(lodashGet(allCountries, value, ''));
+    }, [value, allCountries]);
```

Comment:
Please apply same fix for the StatePicker

Reply:
Has this been resolved? @koko57 @s77rt

Reply:
@mountiny Yes